packages/service-library/src/servicelib/pools.py

```diff
@@ -0,0 +1,34 @@
+from concurrent.futures import ProcessPoolExecutor
+from contextlib import contextmanager
+
+# only gets created on use and is guaranteed to be the s
+# ame for the entire lifetime of the application
+__shared_process_pool_executor = None
```

Comment:
Not a huge fan of globals but I guess here is justified.

Reply:
Yes, I know what you mean.